src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -413,7 +413,8 @@
 							  Grid.Column="2"
 							  Width="40"
 							  Height="40"
-							  HorizontalAlignment="Left">
+							  HorizontalAlignment="Left"
+							  VerticalAlignment="Center">
```

Comment:
Would it be interesting to have a validation in the sample app?

Reply:
I'm sure, but visually it would be interesting to see anyways.

Reply:
It's easy to visualise, in Uno.Themes\src\samples\Uno.Themes.Samples\Uno.Themes.Samples.Shared\Content\Controls\ToggleSwitchSamplePage.xaml line 110 
set the stack panel orientation horizontal :
![image](https://user-images.githubusercontent.com/888391/193125001-a51e4b22-d6fc-48c8-9495-110e19200be6.png)